lib/bing/ads/api/v13/services/base.rb

```diff
@@ -73,6 +73,14 @@ def call(operation, payload)
               rescue Savon::InvalidResponseError => error
                 # TODO better handling
                 raise
+              rescue Bing::Ads::API::Errors::RateLimitError => error
+                if retries_made < retry_attempts
+                  # https://learn.microsoft.com/en-us/advertising/guides/handle-service-errors-exceptions?view=bingads-13#code-117
+                  sleep(90 * (2**retries_made))
```

Comment:
why this logic? isnt it saying to sleep for 60 seconds? so a linear backoff like (60sec + randome jitter bounded by 10 sec or so) should be good enough - no?